src/zenml/zen_stores/schemas/curated_visualization_schemas.py

```diff
@@ -0,0 +1,241 @@
+    CuratedVisualizationResponseResources,
+    CuratedVisualizationUpdate,
+)
+from zenml.zen_stores.schemas.base_schemas import BaseSchema
+from zenml.zen_stores.schemas.project_schemas import ProjectSchema
+from zenml.zen_stores.schemas.schema_utils import (
+    build_foreign_key_field,
+    build_index,
+)
+from zenml.zen_stores.schemas.utils import jl_arg
+
+if TYPE_CHECKING:
+    from zenml.zen_stores.schemas.artifact_visualization_schemas import (
+        ArtifactVisualizationSchema,
+    )
+
+
+class CuratedVisualizationSchema(BaseSchema, table=True):
+    """SQL Model for curated visualizations."""
+
+    __tablename__ = "curated_visualization"
+    __table_args__ = (
+        build_index(__tablename__, ["artifact_visualization_id"]),
+        UniqueConstraint(
+            "artifact_visualization_id",
+            "resource_id",
+            "resource_type",
+            name="unique_curated_visualization_resource_link",
+        ),
+    )
+
+    project_id: UUID = build_foreign_key_field(
+        source=__tablename__,
+        target=ProjectSchema.__tablename__,
+        source_column="project_id",
+        target_column="id",
+        ondelete="CASCADE",
+        nullable=False,
+    )
+    artifact_visualization_id: UUID = build_foreign_key_field(
+        source=__tablename__,
+        target="artifact_visualization",
+        source_column="artifact_visualization_id",
+        target_column="id",
+        ondelete="CASCADE",
+        nullable=False,
+    )
+
+    display_name: Optional[str] = Field(default=None)
+    display_order: Optional[int] = Field(default=None)
+    layout_size: str = Field(
+        default=CuratedVisualizationSize.FULL_WIDTH.value,
+        nullable=False,
+    )
+    resource_id: UUID = Field(nullable=False)
+    resource_type: str = Field(nullable=False)
+
+    artifact_visualization: "ArtifactVisualizationSchema" = Relationship(
+        back_populates="curated_visualizations"
+    )
+
+    @classmethod
+    def get_query_options(
+        cls,
+        include_metadata: bool = False,
+        include_resources: bool = False,
+        **kwargs: Any,
+    ) -> Sequence[ExecutableOption]:
+        """Get the query options for the schema.
+
+        Args:
+            include_metadata: Whether metadata will be included when converting
+                the schema to a model.
+            include_resources: Whether resources will be included when
+                converting the schema to a model.
+            **kwargs: Keyword arguments to allow schema specific logic
+
+        Returns:
+            A list of query options.
+        """
+        options: List[ExecutableOption] = []
+
+        if include_resources:
+            from zenml.zen_stores.schemas.artifact_visualization_schemas import (
+                ArtifactVisualizationSchema,
+            )
+
+            options.append(
+                selectinload(jl_arg(cls.artifact_visualization)).options(
+                    selectinload(
```

Comment:
The second one isn't necessary, as it won't be included in the visualization response.